packages/terra-progress-bar/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Added new colorClass prop and examples of how to use it.
+
+### Removed
```

Comment:
This should be listed under a heading "Breaking Change".

Reply:
Fixed now. https://github.com/cerner/terra-core/pull/2211/commits/e688373e7c8eddfa09f8e8de1fa4e9ef8750426e